pom.xml

```diff
@@ -67,6 +67,7 @@
   <properties>
     <mavenVersion>3.1.0</mavenVersion>
     <maven.compiler.release>8</maven.compiler.release>
+    <maven.compiler.target>8</maven.compiler.target> <!-- this property is needed by enforcer plugin configured in maven-parent -->
```

Comment:
I had to insert this property because enforcer-maven-plugin configured in maven-parent uses this property. Without this property I got following error:

```
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-bytecode-version) @ maven-jlink-plugin ---
[INFO] Adding ignore: module-info
[INFO] Restricted to JDK 1.7 yet org.mockito:mockito-core:jar:3.5.13:test contains org/mockito/AdditionalAnswers.class targeted to JDK 8
[INFO] Restricted to JDK 1.7 yet org.junit.platform:junit-platform-commons:jar:1.7.0:test contains org/junit/platform/commons/JUnitException.class targeted to JDK 8
[INFO] Restricted to JDK 1.7 yet org.objenesis:objenesis:jar:3.1:test contains org/objenesis/ObjenesisHelper.class targeted to JDK 8
[INFO] Restricted to JDK 1.7 yet org.junit.vintage:junit-vintage-engine:jar:5.7.0:test contains org/junit/vintage/engine/JUnit4VersionCheck.class targeted to JDK 8
[INFO] Restricted to JDK 1.7 yet org.assertj:assertj-core:jar:3.16.1:test contains org/assertj/core/condition/Not.class targeted to JDK 8
[INFO] Restricted to JDK 1.7 yet org.junit.jupiter:junit-jupiter-params:jar:5.7.0:test contains org/junit/jupiter/params/ParameterizedTest.class targeted to JDK 8
[INFO] Restricted to JDK 1.7 yet org.junit.jupiter:junit-jupiter-engine:jar:5.7.0:test contains org/junit/jupiter/engine/Constants.class targeted to JDK 8
[INFO] Restricted to JDK 1.7 yet org.junit.jupiter:junit-jupiter-api:jar:5.7.0:test contains org/junit/jupiter/api/AfterAll.class targeted to JDK 8
[INFO] Restricted to JDK 1.7 yet org.junit.platform:junit-platform-engine:jar:1.7.0:test contains org/junit/platform/engine/CompositeFilter$1.class targeted to JDK 8
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.EnforceBytecodeVersion failed with message:
Found Banned Dependency: org.mockito:mockito-core:jar:3.5.13
Found Banned Dependency: org.junit.platform:junit-platform-commons:jar:1.7.0
Found Banned Dependency: org.objenesis:objenesis:jar:3.1
Found Banned Dependency: org.junit.vintage:junit-vintage-engine:jar:5.7.0
Found Banned Dependency: org.assertj:assertj-core:jar:3.16.1
Found Banned Dependency: org.junit.jupiter:junit-jupiter-params:jar:5.7.0
Found Banned Dependency: org.junit.jupiter:junit-jupiter-engine:jar:5.7.0
Found Banned Dependency: org.junit.jupiter:junit-jupiter-api:jar:5.7.0
Found Banned Dependency: org.junit.platform:junit-platform-engine:jar:1.7.0
Use 'mvn dependency:tree' to locate the source of the banned dependencies.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.723 s
[INFO] Finished at: 2020-10-29T10:05:48+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M3:enforce (enforce-bytecode-version) on project maven-jlink-plugin: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed. -> [Help 1]
```

Reply:
Fix see next commit